docs/bambu_tutorial.ipynb

```diff
@@ -0,0 +1,746 @@
+            "\n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "if (!requireNamespace(\"BiocManager\", quietly = TRUE))\n",
+        "    install.packages(\"BiocManager\")\n",
+        "\n",
+        "BiocManager::install(\"bambu\", update = FALSE)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jEShAjw3ktnI"
+      },
+      "source": [
+        "If you want a more recent version of Bambu, you may refer to the Bambu Github repository [here](https://github.com/GoekeLab/bambu). "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "uOVH1uPGgLZ6"
+      },
+      "source": [
+        "\n",
+        "## **Data Access and Preparation**\n",
+        "### **Download Data for Bambu**\n",
+        "Next, we will need to download the required data to run Bambu. The required data include:\n",
+        "\n",
+        "-   a set of aligned reads to the genome from the A549 and HepG2 cell lines (bam files),\n",
+        "-   reference human genome annotations (gtf file, TxDb object, or Bambu\n",
+        "    Annotation object),\n",
+        "-   reference human genome sequence (fasta file or BSgenome).\n",
+        "\n",
+        "Generally, you may want to learn how to get access to these data using the [data\n",
+        "access\n",
+        "tutorial](AWS_data_access_tutorial.md). Below we only show the necessary steps to download the required data. The following command requires you to have [AWS CLI](https://aws.amazon.com/cli/) installed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "NPdFZL70YT-D",
+        "vscode": {
+          "languageId": "r"
+        }
+      },
+      "outputs": [],
+      "source": [
+        "# install AWS CLI\n",
+        "system(\"sudo apt install awscli\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "drRIHMvfXW15",
+        "vscode": {
+          "languageId": "r"
+        }
+      },
+      "outputs": [],
+      "source": [
+        "# create a directory to store the data\n",
+        "system(\"mkdir bambu_tutorial\")\n",
+        "# download genome fasta file \n",
+        "system(\"aws s3 cp --no-sign-request s3://sg-nex-data/data/data_tutorial/annotations/hg38_chr22.fa ./bambu_tutorial\")\n",
+        "# download genome index fastai file \n",
+        "system(\"aws s3 cp --no-sign-request s3://sg-nex-data/data/data_tutorial/annotations/hg38_chr22.fa.fai ./bambu_tutorial\")\n",
+        "# download gtf file\n",
+        "system(\"aws s3 cp --no-sign-request s3://sg-nex-data/data/data_tutorial/annotations/hg38_chr22.gtf ./bambu_tutorial\")\n",
+        "# download aligned bam files for A549 samples and HepG2 samples\n",
```

Comment:
these lines seem to be too long, they are not displayed completely anymore. maybe they can be split into 2 lines?